block.go

```diff
@@ -288,6 +309,17 @@ func OpenBlock(dir string, pool chunkenc.Pool) (*Block, error) {
 	return pb, nil
 }
 
+func blockSize(rr ...SizeReader) int64 {
+	var total int64
+	for _, r := range rr {
+		if r != nil {
+			t := r.Size()
+			total += t
```

Comment:
We can directly do it as `total += r.Size()` instead of having an extra variable declaration.